chore/develop-to-master

Ricardo385 opened the pull request:
chore/develo-to-master

Ricardo385 committed:
docs(): README updated

Ricardo385 committed:
Merge pull request #65 from openforge/docs/readme-updated

docs(): README updated

jreusser commented on README.md:
```diff
@@ -1,3 +1,16 @@
+<p align="center">
+  <img src="https://s3.amazonaws.com/tw-inlineimages/467579/0/0/ff47c124a5732bf549b43532a7ac19e1.png"/>
```
are there appropriate external host restrictions around serving this file?

jreusser commented:
> chore/develo-to-master

this should ideally be 'develop', but that's such a minor gripe

emmanuel-Andela committed:
feature/UpdateDependencies
- Updated NX
- Updated child angular project
- Updated react child project
- Updated vue child project
- Updated vitest
- Improved lint stagged
- Upgraded capacitor
- Updated as much as possible eslint and stylelint libs

jedihacks committed:
Merge pull request #69 from openforge/feature/UpdateDependencies

feature/UpdateDependencies
